Snippets/ASP/ASTP_5/Usage.cs

```diff
@@ -0,0 +1,193 @@
+
+        persistence.JsonSettings(new JsonSerializerSettings
+        {
+            Converters =
+                {
+                    new IsoDateTimeConverter
+                    {
+                        DateTimeStyles = DateTimeStyles.RoundtripKind
+                    }
+                }
+        });
+        #endregion
+    }
+
+    public void ConfigureCustomJsonReader(EndpointConfiguration endpointConfiguration)
+    {
+        #region AzurePersistenceSagasReaderCreator
+        var persistence = endpointConfiguration.UsePersistence<AzureTablePersistence, StorageType.Sagas>();
+
+        persistence.ReaderCreator(
+            readerCreator: textReader =>
+            {
+                return new JsonTextReader(textReader);
+            });
+        #endregion
+    }
+
+    public void ConfigureCustomJsonWriter(EndpointConfiguration endpointConfiguration)
+    {
+        #region AzurePersistenceSagasWriterCreator
+
+        var persistence = endpointConfiguration.UsePersistence<AzureTablePersistence, StorageType.Sagas>();
+
+        persistence.WriterCreator(
+            writerCreator: writer =>
+            {
+                return new JsonTextWriter(writer)
+                {
+                    Formatting = Formatting.None
+                };
+            });
+
+        #endregion
+    }
+
+    void CustomizingDefaultTableName(EndpointConfiguration endpointConfiguration)
+    {
+        #region SetDefaultTable
+
+        var persistence = endpointConfiguration.UsePersistence<AzureTablePersistence>();
+        persistence.ConnectionString("connectionString");
+        persistence.DefaultTable("TableName");
+
+        endpointConfiguration.EnableInstallers();
+
+        #endregion
+    }
+
+    void EnableInstallersConfiguration(EndpointConfiguration endpointConfiguration)
+    {
+        #region EnableInstallersConfiguration
+
+        var persistence = endpointConfiguration.UsePersistence<AzureTablePersistence>();
+        persistence.ConnectionString("connectionString");
+        persistence.DefaultTable("TableName");
+
+        endpointConfiguration.EnableInstallers();
+
+        #endregion
+    }
+
+    void EnableInstallersConfigurationOptingOutFromTableCreation(EndpointConfiguration endpointConfiguration)
+    {
+        #region EnableInstallersConfigurationOptingOutFromTableCreation
+
+        var persistence = endpointConfiguration.UsePersistence<AzureTablePersistence>();
+        persistence.ConnectionString("connectionString");
+        persistence.DefaultTable("TableName");
+
+        // make sure the table name specified in the DefaultTable exists when calling DisableTableCreation
+        endpointConfiguration.EnableInstallers();
+        persistence.DisableTableCreation();
+
+        #endregion
+    }
+
+    void ConfigureThrottlingSettingsOnStorageTableClient(EndpointConfiguration endpointConfiguration, TableServiceClient tableServiceClient)
```

Comment:
Is this snippet a wip? The text in `index_ratelimiting_astp_[5,).partial.md` suggests the retry policy can be adjusted by using `TableClientOptions`, and it looks like that was started but not finished?

Reply:
Good catch Andreas, this just needs to go, as it's part of the SDK api.